src/components/Header.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <nav>
+    <ul>
+      <li>
+        <router-link to="/" exact>Home</router-link>
```

Comment:
Koristila bih imena sami route-a
`<router-link :to="{ name: 'TodoList' }"`>
Ukoliko kasnije budes menjao strukturu URL-a. :)